INSTALL

```diff
@@ -328,6 +328,42 @@ have a look at the following Xen toolstack:
 	http://github.com/cnplab/cosmos
 
 
+DPDK
+----
+Support for using Intel DPDK for high speed userspace I/O is enabled using
+
+    ./configure --enable-dpdk --enable-user-multithread --disable-linuxmodule
+
+In order to configure and compile properly, the RTE_SDK and RTE_TARGET
+environment variable must be set, e.g.
+
+    export RTE_SDK=/path/to/dpdk-2.1.0/
+    export RTE_TARGET=x86_64-native-linuxapp-gcc
```

Comment:
I think you should suggest instead

```
./configure RTE_SDK=... RTE_TARGET=... --enable-dpdk --enable-user-multithread
```

since then ./config.status will remember the settings.